src/match.js

```diff
@@ -1,19 +1,24 @@
+var _curry2 = require('./internal/_curry2');
+var compose = require('./internal/compose');
+var defaultTo = require('./defaultTo');
 var invoker = require('./invoker');
 
 
 /**
- * Tests a regular expression against a String
+ * Tests a regular expression against a String. Note that this function
+ * will return an empty array when there are no matches. This differs
+ * from the standard JavaScript string function which returns null when
```

Comment:
Let's replace

    the standard JavaScript string function

with

    [`String.prototype.match`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/match)